cmd/root.go

```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/99designs/gqlgen/graphql"
 	"github.com/99designs/gqlgen/internal/gopath"
 	"github.com/urfave/cli"
+
+	// Required since otherwise dep will prune away these unused packages before codegen has a chance to run
+	_ "github.com/99designs/gqlgen/handler"
```

Comment:
This seems to be enough to get `init` to run cleanly.  Attempting to run `server/server.go` afterwards however still requires some additional deps (websockets I think).  So I added a `dep ensure` after `gqlgen init` in the docs.